cilium-health/cmd/root.go

```diff
@@ -74,13 +76,14 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	flags := rootCmd.PersistentFlags()
 	flags.StringP("admin", "", string(serverPkg.AdminOptionUnix), "Expose resources over 'unix' socket, 'any' socket")
+	flags.StringP("cilium", "c", "", "URI to Cilium server API")
 	flags.BoolP("debug", "D", false, "Enable debug messages")
 	flags.BoolP("daemon", "d", false, "Run as a daemon")
-	flags.BoolP("passive", "p", false, "Only respond to HTTP health checks")
```

Comment:
Why are "passive" and "pidfile" removed?

Reply:
Sorry, it was somehow removed during cherry-picking. I will add them again